README.md

```diff
@@ -44,6 +44,51 @@ custom:
     - SomeOtherKeyThatAssignsRandomValue
 ```
 
+### Specifying usage plan
+
+```yaml
+custom:
+  apiKeys:
+    - name: SomeKey
+      usagePlan:
+        name: "name-of-first-usage-plan" (required if usagePlan is specified. rest of the fields are optional)
+        description: "Description of first plan"
+        quota:
+          limit: 1000
+          period: DAY
+        throttle:
+          burstLimit: 100
+          rateLimit: 20
+    - name: SomeOtherKey
+      usagePlan:
+        name: "name-of-first-usage-plan"
+    - name: ThirdKey
+      usagePlan:
+        name: "name-of-second-plan"
+        description: "Description of second plan"
+        quota:
+          limit: 2000
+          period: DAY
+        throttle:
+          burstLimit: 100
+          rateLimit: 20
+    - name: AKeyWithNoUsagePlan
+provider:
+  usagePlan:
```

Comment:
@christopherchubb  rather than adding `usagePlan` under `provider` section, i think we should keep it under `custom` section like `apiKeys`.. thoughts?

Reply:
provider::usagePlan is already defined by serverless. It is used if the plugin is not used or no usagePlans are defined.